www/src/pages/blog/astro-018.astro

```diff
@@ -0,0 +1,112 @@
+---
+import { Markdown, Prism } from 'astro/components';
+import BlogHead from '../../components/BlogHead.astro';
+import BlogHeader from '../../components/BlogHeader.astro';
+import BlogPost from '../../components/BlogPost.astro';
+import BlockQuote from '../../components/BlockQuote.astro';
+import GoogleAnalytics from '../../components/GoogleAnalytics.astro';
+import Note from '../../components/Note.astro';
+import Shell from '../../components/Shell.astro';
+
+let title = 'Astro 0.18 Release';
+let description = `Astro 0.18 is our biggest release since Astro launch. It includes a new responsive hydrator, named slots, global stylesheets, and two new renderers. Ready on to learn more on this release.`;
+let publishDate = 'Tuesday, July 27 2021';
+let author = 'matthew';
+let heroImage = '/social.jpg';
+let permalink = 'https://astro.build/blog/astro-0-18';
+let lang = 'en';
+---
+
+<html lang={ lang ?? 'en' }>
+  <head>
+    <BlogHead title={title} description={description} permalink={permalink} />
+    <style>
+      img {
+        max-width: 100%;
+      }
+    </style>
+  </head>
+
+  <body>
+    <BlogHeader />
+    <BlogPost title={title} author={author} heroImage={heroImage} publishDate={publishDate}>
+        <Markdown>
+          After the success of Astro's launch we spent some time fixing bugs and taking user feedback on what to tackle next. Now some of the features that people requested are starting to get released.
+          
+          Astro 0.18 is the culmination and is the biggest release to date. This post highlights only some of what it brings.
+
+          ![Code example that shows off using the new client:media hydrator.](/assets/blog/astro-018/responsive-hydration.png)
+          <!-- Saved from https://carbon.now.sh/16xchqPVdt5IEAY3Czg3 -->
+
+          ## Responsive component hydration
+
+          Astro uses [partial hydration](https://docs.astro.build/core-concepts/component-hydration), a technique to only include the client-side JavaScript for the parts of a page that are actually interactive. With Astro there were previously 3 strategies for partial hydration: hydrating on [page load](https://docs.astro.build/core-concepts/component-hydration#mycomponent-clientload-), when the CPU is [idle](https://docs.astro.build/core-concepts/component-hydration#mycomponent-clientidle-), or when the element is scrolled into [view](https://docs.astro.build/core-concepts/component-hydration#mycomponent-clientvisible-).
```

Comment:
this is a pretty heavy paragraph to put first, and doesn't really grab your attention. What about reordering to put the next paragraph first? Or even better, what if you merge this info into the third paragraph:

>  When developing responsive sites you often have components that are only visible on certain device sizes, like a mobile sidebar menu. Most developers don't have a good way to load mobile-only code on mobile devices, and end up wasting valuable performance loading code in the browser that will never even be seen by the end user.
> 
> Astro solves this problem using partial hydration, and v0.18 adds a new, 4th way to hydrate components using a **[CSS media query](https://docs.astro.build/core-concepts/component-hydration#mycomponent-clientmediaquery-)**. Add the `client:media` directive to a component and it will only hydrate once the media condition is met.